target/linux/realtek/image/Makefile

```diff
@@ -54,17 +61,29 @@ define Device/d-link_dgs-1210-28
 endef
 TARGET_DEVICES += d-link_dgs-1210-28
 
-define Device/netgear_gs110tpp-v1
-  $(Device/Default)
+# "NGE" refers to the uImage magic
+define Device/Netgear_NGE
+  $(Device/Default_LZMA)
   SOC := rtl8380
   IMAGE_SIZE := 14848k
   UIMAGE_MAGIC := 0x4e474520
-  DEVICE_VENDOR := NETGEAR
-  DEVICE_MODEL := GS110TP
+  DEVICE_VENDOR := Netgear
+endef
+
+define Device/netgear_gs110tpp-v1
+  $(Device/Netgear_NGE)
```

Comment:
Could Device/Netgear_NGE be added in the first commit? Then the second commit can contain everything to modify the GS110TPP. The last commit only adds GS108Tv3 support, and nothing else.

Reply:
Sure.  @adschm has left some comments above which I will address in a new commit.  Once there are no further changes requested I will force-push a new commit stack as described.